tdrs-backend/tdpservice/users/test/test_api.py

```diff
@@ -2,6 +2,7 @@
 from django.contrib.auth import get_user_model
```

Comment:
After adding the STT to the user JSON response, all of the tests needed to be updated to also check that the STT is returning properly

Reply:
Looks good!

Reply:
Revisiting this file, I do have a question about `def test_retrieve_user`, even though it isn't in the change set:

```
def test_retrieve_user(api_client, user):
    """Test user retrieval."""
    response = api_client.get(f"/v1/users/{user.pk}/")
    assert response.status_code == status.HTTP_200_OK
    assert response.data["username"] == user.username
```

I noticed that this test doesn't have an `api_client.login` line. Does this mean that anyone, authenticated or not, can retrieve a user's information via the API?

Reply:
The endpoint this is testing was originally created for the front end to call after login. We opted to go with returning that data with the `authorization-check` endpoint when the user is authorized, so as to not require an additional call. However, in the future, this endpoint will be needed for wider use in the application itself, so while it's not entirely completed, it didn't make sense to remove the work just to repeat it later on.

When that requirement hits, we'll make sure a user is authorized and has the proper permissions to access the data. Since permissions are not yet implemented, it makes more sense to tackle that then. This simply tests that the endpoint is working properly as created.